.travis.yml

```diff
@@ -9,5 +9,5 @@ script:
   - npm run coverage
   - npm run build
 after_success:
-  - npm install -g coveralls
-  - cat coverage/lcov.info | coveralls
+  - npm install -g codecov
```

Comment:
Does Jest not have this built in? Or do we need this for Travis?

Reply:
This is for travis.